packages/builder/src/components/backend/DatasourceNavigator/icons/index.js

```diff
@@ -44,6 +44,9 @@ export default ICONS
 
 export function getIcon(integrationType, schema) {
   const integrationList = get(integrations)
+  if (!integrationList) {
```

Comment:
Why is this supported? Should the integration not be required?

Reply:
I ran into an issue with the `integrationList` not being loaded initially when this was called and bricking the app due to failing to retrieve anything, just wanted to avoid this scenario going forward!